apps/job-info-consumer/Makefile

```diff
@@ -0,0 +1,51 @@
+.PHONY: all
```

Comment:
It would be nice to have directives in the top level Makefile of the project to build (and test!) this package when the command `make all` is issued.